src/daemon/lldpd.c

```diff
@@ -170,6 +170,9 @@ lldpd_alloc_default_local_port(struct lldpd *cfg)
 		calloc(1, sizeof(struct lldpd_port))) == NULL)
 		fatal("main", NULL);
 
+	/* Initialize TX VLAN */
```

Comment:
Is this the correct place to initialize port->p_vlan_tx_id?

Reply:
Yes, it looks good.